src/sentry/conf/server.py

```diff
@@ -1665,18 +1665,15 @@ def create_partitioned_queues(name):
 )
 
 
-def get_sentry_sdk_config():
-    return {
-        "release": sentry.__build__,
-        "environment": ENVIRONMENT,
-        "in_app_include": ["sentry", "sentry_plugins"],
-        "_experiments": {"smart_transaction_trimming": True},
-        "debug": True,
-        "send_default_pii": True,
-    }
-
-
-SENTRY_SDK_CONFIG = get_sentry_sdk_config()
+SENTRY_SDK_CONFIG = {
```

Comment:
Why do we need this change and why do we keep that method around if we're not gonna use it anymore?

Reply:
We don't need this change, it just used to be that the purpose of get_sentry_sdk_config was to avoid polluting the global namespace, however nowadays it does not contain any local variables.

Reply:
>why do we keep that method around if we're not gonna use it anymore?

the method is gone though